guide/yaml/yaml-reference.md

```diff
@@ -213,6 +213,9 @@ concise DSL defined here:
   `$brooklyn:root()`.
 * `$brooklyn:formatString("pattern e.g. %s %s", "field 1", "field 2")` returns a future which creates the formatted string
   with the given parameters, where parameters may be strings *or* other tasks such as `attributeWhenReady`
+* `$brooklyn:urlEncode("val")` returns a future which creates a string with the characters escaped
```

Comment:
I'm not sure it's worth using a term like 'future' here which the user may not be aware of. From their perspective this function just URL Encodes the parameter?